bazel/hephaestus.bzl

```diff
@@ -107,6 +107,11 @@ def heph_cc_binary(
     cc_binary(
         copts = heph_copts() + copts,
         linkopts = heph_linkopts() + linkopts,
+        env = {
+          # Leak detection currently doesn't work due to zenoh
+          "ASAN_OPTIONS": "detect_leaks=0",
+          "UBSAN_OPTIONS": "print_stacktrace=1:halt_on_error=1",
```

Comment:
does this have an impact on performance?

Reply:
The env variables are ignored on non sanitizer builds. In general, sanitizers add a performance overhead, yes.